tests/cli/test_cmd_rm.py

```diff
@@ -31,14 +31,14 @@
 
 RM_CMD_OP = """mutation {{
   deleteIdentity(uuid: "{}") {{
-    uidentity {{
+    individual {{
       uuid
     }}
   }}
 }}"""
 
 RM_UID_OUTPUT = (
```

Comment:
Maybe it is not important, but the name of this output could be renamed to `RM_INDV_OUTPUT` (or similar).

Reply:
This shouldn't be renamed because `rm` removes identities and individuals.